conf/logback-test.xml

```diff
@@ -1,16 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
 
+    <conversionRule conversionWord="simpleMethodLog" converterClass="utils.logback.SimpleMethodLog"/>
+
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
         <file>logs/income-tax-view-change-frontend.log</file>
         <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
-            <pattern>[%.-4level]  %date{ISO8601} %message %replace(exception=[%xException]){'^exception=\[\]$',''}%n</pattern>
+            <pattern>%date{ISO8601} %level %simpleMethodLog [%message] %replace(exception=[%xException]){'^exception=\[\]$',''} thread=[%thread] %n</pattern>
+            <!--
+                <pattern>[%.-4level]  %date{ISO8601} %message %replace(exception=[%xException]){'^exception=\[\]$',''}%n</pattern>
+             -->
```

Comment:
default directives add more noise like $ characters and method names (mentioned in the tickets) etc, this custom approach gives us a cleaner output that's easier to discern, happy to discuss and change

Reply:
Hmm... I'm always wary of hand-crafting solutions like this. Does this account for all the scenarios possible? For example, Java anonymous classes have $ in them too, and are required to distinguish them from their containing classes. The Scala compiler does even more mangling of the source to get it into a form that the jvm will run. Are we sure we're not loosing anything here?

When I was discussing this as part of the investigation ticket, I flagged this up more as a warning to the team that we would be getting this noise with the logging in this manner so we could get agreement that it was worth doing this way, rather than suggesting we try to get rid of it. Maybe I could have made that clearer.